src/components/VmDialog/index.js

```diff
@@ -347,6 +351,7 @@ class VmDialog extends React.Component {
             onChange={this.onChangeCluster}
             value={cluster ? cluster.get('id') : ''}
             data={sortedClusters}
+            fieldHelp={<FieldHelp title='Cluster' content='Group of servers the virtual machine can be running on.' />}
```

Comment:
I would use the term "hosts" instead of "servers" to be aligned with the webadmin terminology